go/teams/implicit.go

```diff
@@ -60,34 +60,34 @@ func LookupImplicitTeamAndConflicts(ctx context.Context, g *libkb.GlobalContext,
 
 func LookupImplicitTeamIDUntrusted(ctx context.Context, g *libkb.GlobalContext, displayName string,
 	public bool) (res keybase1.TeamID, err error) {
-	imp, err := loadImpteam(ctx, g, displayName, public)
+	imp, _, err := loadImpteam(ctx, g, displayName, public, false /* force */)
 	if err != nil {
 		return res, err
 	}
 	return imp.TeamID, nil
 }
 
-func loadImpteam(ctx context.Context, g *libkb.GlobalContext, displayName string, public bool) (imp implicitTeam, err error) {
+func loadImpteam(ctx context.Context, g *libkb.GlobalContext, displayName string, public bool, force bool) (imp implicitTeam, hitCache bool, err error) {
```

Comment:
maybe `forceCache` or `useCache` instead? i feel like `force` would typically imply forcing a server call not a cache call so a more explicit name might be less confusing in the future